Scripts/Weapon/DashSkill.cs

```diff
@@ -244,6 +244,7 @@ protected override void OnAttackButtonReleased()
         isRunning = true;
         dashLeft = duration;
         Vector3 direction = (targetLocation - currentLocation).Normalized();
+        direction.y = 0;

```

Comment:
Or if we want the enemy to fly up a bit when hit by the skill, we can change this to a non-zero value.

Reply:
Ya, if not maybe the attack looks too weak?

Reply:
Ok I'll try with some values